api/lanes.py

```diff
@@ -581,7 +583,6 @@ def lane_query_hook(self, qu, **kwargs):
         # regardless of work_model and existing joins.
         work_edition = aliased(Edition)
         qu = qu.join(work_edition).filter(work_edition.series==self.series)
-        qu = qu.order_by(work_edition.series_position, work_edition.title)
```

Comment:
I removed the series position sort from the series lane query, since it's a facet now.